packages/palette/src/elements/Button/Button.ios.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react"
 import { TouchableWithoutFeedback } from "react-native"
-import { animated, Spring } from "react-spring/dist/native"
+import { animated, Spring } from "react-spring/renderprops-native.cjs"
```

Comment:
@l2succes - we should be using hooks for animation now, but in the case of porting over code this is the new import path in `react-spring` 8.x.

Reply:
Got it, sounds good @damassi